# Goal
string qr_tag
---
# Result
uint8 state
---
# Feedback
float32 distance_to_goal
